apps/storybook/src/Selection.stories.tsx

```diff
@@ -173,7 +173,9 @@ export const Persisted: Story<TemplateProps> = (args) => {
         onSelectionEnd={setPersistedSelection}
         modifierKey={selectionModifierKey}
       >
-        {(selection) => <SelectionComponent {...selection} />}
+        {(selection) =>
+          !persistedSelection && <SelectionComponent {...selection} />
```

Comment:
1. `onSelectionEnd` is called with the selection to persist.
2. The story re-renders with a persisted selection => **both the persisted and active selections are visible at once**
3. The internal active selection state of `SelectionTool` is cleared
4. `SelectionTool` re-renders and returns `null` because there's no longer an active selection.

I couldn't find an easy to way for `SelectionTool` to re-render and return `null` _before_ `onSelectionEnd` gets called, so instead I work around the issue with a simple condition in the render prop.

Reply:
Not a huge problem but good to be aware of it